src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Card/BitCard.scss

```
@@ -1,182 +1,533 @@
+    @if $hover > 24 { $hover: 24; }
+
+    $active: $i - 2;
+    @if $active < 1 { $active: 0; }
+
+    .bit-crd-e#{$i} {
+        --bit-crd-shd: var(--bit-shd-#{$i});
+        --bit-crd-shd-hov: var(--bit-shd-#{$hover});
+
+        @if $active == 0 {
+            --bit-crd-shd-act: none;
+        }
```

Stylelint (17.14.0)

[error] Unexpected newline after "}" of `@if` statement (scss/at-if-closing-brace-newline-after)

(scss/at-if-closing-brace-newline-after)